frontend/components/selector/SelectorTopBar.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <div v-if="selectors.length > 0" class="w-full h-fit">
+    <div class="flex flex-row"> 
+      <button
+        v-for="t in selectors"
+        class="w-full justify-center bg-light-header dark:bg-dark-header flex px-3 py-1 border-l-[1px] border-y-[1px] border-light-text dark:border-dark-text text-light-text dark:text-dark-text hover:bg-black/10 dark:hover:bg-white/30"
+      >
+        {{ t }}
+      </button>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+defineProps<{
+  selectors?: string[];
+}>();
+</script>
```

Comment:
For here we need a blank line at the end of the file, @sayik :) I'd set up VS Code to always add one via your settings.json file 🙃